pose_est_nets/utils/plotting_utils.py

```diff
@@ -445,7 +474,7 @@ def make_predictions(
     # build dataframe from the hierarchal index and predictions array
     df = pd.DataFrame(predictions, columns=pd_index)
 
-    return df
+    return df, heatmaps_np
```

Comment:
how does returning `heatmaps_np` play out with the regression model? did you try running this func? just so it doesn't break things?

Reply:
I think it should work fine for regression tracker, we set the heatmaps_np to none if its regression model and we only try to save the heatmaps if they are not none. I can test this later when I start up an instance to do some other stuff.